setup.py

```diff
@@ -0,0 +1,30 @@
+import pathlib

```

Comment:
Can you add 
1. A pylama check for this whole package
2. CI check using Travis CI or github actions, or whatever works for you
3. A badge in the readme indicating if the current `main` is passing the CI test?

Reply:
Working on it